src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -606,11 +600,27 @@ private static string CreateDescription(OperationGroup operationGroup, string cl
                 BuilderHelpers.EscapeXmlDescription(operationGroup.Language.Default.Description);
         }
 
-        private static string CreateDescription(RequestParameter requestParameter)
+        private static string CreateDescription(RequestParameter requestParameter, CSharpType type)
         {
-            return string.IsNullOrWhiteSpace(requestParameter.Language.Default.Description) ?
+            var description = string.IsNullOrWhiteSpace(requestParameter.Language.Default.Description) ?
                 $"The {requestParameter.Schema.Name} to use." :
                 BuilderHelpers.EscapeXmlDescription(requestParameter.Language.Default.Description);
+
+            return requestParameter.Schema switch
+            {
+                ChoiceSchema choiceSchema when type.IsFrameworkType => AddAllowedValues(description, choiceSchema.Choices),
+                SealedChoiceSchema sealedChoiceSchema when type.IsFrameworkType => AddAllowedValues(description, sealedChoiceSchema.Choices),
+                _ => description
+            };
+
+            static string AddAllowedValues(string description, ICollection<ChoiceValue> choices)
```

Comment:
If allowed enum values are stored in the parameter description, they will be written everywhere the parameter is used.